UI/ChatGPT/ChatGPT/ChatGPT/appsettings.json

```diff
@@ -1,7 +1,7 @@
 {
   "AppConfig": {
     "Environment": "Production",
-    "ApiKey": "<Insert your OpenAI API key here. Access https://platform.openai.com/api-keys to generate your OpenAI API key.>"
+    "ApiKey": "" //<Insert your OpenAI API key here. Access https://platform.openai.com/api-keys to generate your OpenAI API key.>
```

Comment:
Json spec does not define a syntax for comments in json. It's commonly accepted to have `//` but this is not valid regarding to the json spec. Did you validated that it works on all platforms?

Reply:
Hmmm... Yeah, it worked on Windows, Skia.GTK, WASM and Android. I'm unable to test iOS at the moment.